creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -829,6 +967,15 @@ class SnapbackSM {
     // This condition will only be hit if the secondary has failed to sync within timeoutMs
     throw new Error(`Secondary ${secondaryUrl} did not sync up to primary for user ${wallet} within ${timeoutMs}ms`)
   }
+
+  async updateEndpointToSpIdMap () {
```

Comment:
the only value i can see to "refreshing" this logic and assigning it to class var is that you could wrap this in a try-catch and if it ever errors you just use the previously assigned class var instead of ending the full current run

Reply:
wrap logic in try/catch, set as instance member variable, use backup if for some reason this call fails